classes/lang/DataLang.php

```diff
@@ -72,6 +76,8 @@ public function __construct($locale, ?TranslatorInterface $translator = null)
                 ->loadLanguage($this->translator, $this->locale);
             $this->translator->getCatalogue($this->locale);
         }
+
+        $this->inflector = InflectorFactory::create()->build();
```

Comment:
No possible way to inject it in the constructor instead of using the factory here?

Reply:
It is possible but it requires too much changes, it is not worth it. Also, doctrine is using it the same way: https://github.com/doctrine/orm/blob/2.9.x/lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php#L82-L86